.cirrus.yml

```diff
@@ -114,3 +114,41 @@ test_windows_task:
   test_erlang_script: make --keep-going test_erlang
 
   test_elixir_script: make --keep-going test_elixir
+
+
+test_freebsd_task:
+  name: FreeBSD 12.0
+
+  freebsd_instance:
+    image_family: freebsd-12-0
+    cpu: 8
+    memory: 7424Mi
```

Comment:
What's your process to get these memory values? (just curious)

Reply:
they are given by the CI. If you go bellow, it will raise an error telling what is the minimum and the maximum for that many CPUs.

Reply:
So do we need to set them at all?

Reply:
I wouldn't be able to tell what's the default value being used
https://cirrus-ci.com/task/5629539488628736

I prefer to go with as little memory as needed, so we can get a slot as quick as possible.

Reply:
> So do we need to set them at all?

@josevalim yes, we do

no cpu , no memory set (default values defined by Cirrus= cpu: 2, memory: 4GB)
finished in 8min 30sec
https://cirrus-ci.com/task/6362284892356608

cpu: 8, it errors.
https://cirrus-ci.com/task/4845801095823360
Failed to start: Invalid value for field 'resource.machineType': 'https://www.googleapis.com/compute/v1/projects/cirrus-ci-community/zones/us-central1-f/machineTypes/custom-8-4096'. Memory size for 8 vCPU instance should be between 7424MiB and 53248MiB, while 4096MiB is requested


cpu: 8, memory: 7424Mi
finished in 6min
https://cirrus-ci.com/task/6338292030636032

cpu: 8, memory: 14848Mi
finished in 6min
https://cirrus-ci.com/task/6343139740090368


So the best option is the original values of cpu: 8, memory: 7424Mi